src/theme/table.scss

```diff
@@ -21,6 +23,18 @@ table .p-row-tooltip [data-pr-tooltip] {
   padding: 0.75rem 0 0.75rem 0.75rem;
 }
 
-.c-table .p-component {
-  font-size: 13px;
+.text-capitalize-from-uppercase {
+  text-transform: lowercase;
+
+  &:first-letter {
+    text-transform: uppercase;
+  }
+}
+
+.column-cut-text {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  text-overflow: ellipsis;
```

Comment:
You can extend this by using scss extends

for example:
```scss
%text-ellipsis {
  text-overflow: ellipsis;
  overflow: hidden;
}

.column-cut-text {
  // ...
  @extend %text-ellipsis;
}
``` 

if this is not compatible with our SCSS version, please use `@mixin` and `@include`